src/main/java/edu/kis/powp/jobs2d/drivers/visitor/DriverCounterVisitor.java

```diff
@@ -0,0 +1,64 @@
+package edu.kis.powp.jobs2d.drivers.visitor;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.DriverMacro;
+import edu.kis.powp.jobs2d.drivers.PreciseLoggerDriver;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+import edu.kis.powp.jobs2d.drivers.adapter.TrackedJob2dDriver;
+import edu.kis.powp.jobs2d.drivers.composite.DriverContainer;
+import edu.kis.powp.jobs2d.drivers.visitor.DriverVisitor;
+import edu.kis.powp.jobs2d.features.driverTransofrmation.TransformingDriver;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class DriverCounterVisitor implements DriverVisitor {
+
+    private int count = 0;
```

Comment:
Please provide a getter. The result of the visitor task should be available outside.

Reply:
Done